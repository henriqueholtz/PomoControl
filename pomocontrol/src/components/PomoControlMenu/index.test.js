import React from 'react';
import { BrowserRouter} from "react-router-dom";
import { render } from '@testing-library/react';
import { PomoControlMenu } from '@pomocontrol-components';

describe('Test Snapshot of Component', () => {
    it('Comparing snapshot PomoControlMenu', () => {
        const { container  } = render(
            <BrowserRouter>
                <PomoControlMenu />
            </BrowserRouter>
        );

        expect(container.firstChild).toMatchSnapshot();
    })
})