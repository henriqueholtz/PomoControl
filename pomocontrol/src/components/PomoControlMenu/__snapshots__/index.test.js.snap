// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshot of Component Comparing snapshot PomoControlMenu 1`] = `
<nav>
  <h1>
    Menu
  </h1>
  <ul>
    <li>
      <a
        href="/dashboard"
      >
        <span>
          Dashboard
        </span>
      </a>
    </li>
    <li>
      <a
        href="/about"
      >
        <span>
          About
        </span>
      </a>
    </li>
    <li>
      <a
        href="/404"
      >
        <span>
          Not Found Custom
        </span>
      </a>
    </li>
  </ul>
</nav>
`;
